TaskLayer/XLSearchTask/XLSearchTask.cs

```diff
@@ -19,11 +19,13 @@ public XLSearchTask() : base(MyTask.XLSearch)
         {
             //Default parameter setting which is different from SearchTask, can be overwriten
             var digestPara = new DigestionParams(
-                minPeptideLength: 5
+                minPeptideLength: 5,
+                maxPeptideLength: 60,
```

Comment:
These are interesting choices. Would you mind adding a comment for why you chose these?